CHANGELOG.md

```diff
@@ -0,0 +1,129 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic
+Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
+
+## [Unreleased]
+
+### Added
+
+- Add Coveralls support
+- Add testing with Tarantool 2.x-latest
+- Add reusable testing workflow (integration testing with latest Tarantool)
+- Add simple CI based on GitHub actions (#114)
+- Add support UUID type in msgpack (#90)
+- Add go modules support (#91)
+- Add utube handling (#85)
+
+### Fixed
+
+- Fix queue tests (#107)
+
+### Changed
+
+- Handle everything with go test (#115)
+- Use plain package instead of module (#134)
+- Make test case consistent with comments (#105)
```

Comment:
It's a fix

Reply:
Updated.

```diff
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -20,12 +20,12 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 ### Fixed
 
 - Fix queue tests (#107)
+- Make test case consistent with comments (#105)
 
 ### Changed
 
 - Handle everything with `go test` (#115)
 - Use plain package instead of module for UUID submodule (#134)
-- Make test case consistent with comments (#105)
 - Reset buffer if its average use size smaller than quater of capacity (#95)
 
 ## [1.5] - 2019-12-29
```